src/win32/findfile.c

```diff
@@ -183,6 +232,12 @@ int git_win32__find_system_dirs(git_str *out, const wchar_t *subdir)
 			&buf, HKEY_CURRENT_USER, REG_MSYSGIT_INSTALL_LOCAL, subdir) && buf.size)
 		git_str_join(out, GIT_PATH_LIST_SEPARATOR, out->ptr, buf.ptr);
 
+#ifdef _WIN64
```

Comment:
Is this defined by mingw also, or just MSVC?

Reply:
What would be the alternative for mingw?

Reply:
Could we write `#if defined(_WIM64) || defined(__MINGW64__)`?

Reply:
I think that we can just do `GIT_ARCH_64` here - this file will only ever be included for Windows.

Reply:
Fixed.